lms/services/course.py

```diff
@@ -53,7 +53,16 @@ def _create(self, authority_provided_id):
         ):
             course_settings.set("canvas", "sections_enabled", False)
 
-        course = Course(
+        return course_settings
+
+    def _create(self, authority_provided_id, name):
+        # By default we'll make our course setting have the same settings
+        # as the application instance
+        course_settings = self.course_settings(
+            self._application_instance, authority_provided_id
+        )
+
+        course = _Course(
```

Comment:
We only create the existing couse and rely on the HGroup.course_group -> grouping_service.curse_grouping to create rows int he new table.